chart/templates/server-deployment.yaml

```diff
@@ -130,14 +154,46 @@ spec:
         {{- if $root.Values.clamav.enabled  }}
         - name: clamav
           image: "{{ $root.Values.clamav.image.value }}:{{ $root.Values.clamav.image.tag }}"
-          imagePullPolicy: {{ $root.Values.image.pullPolicy }}
+          imagePullPolicy: {{ $root.Values.clamav.image.pullPolicy }}
           ports:
-{{ toYaml $root.Values.clamav.ports | indent 12 }}
-{{ toYaml $root.Values.clamav.probes | indent 10 }}
+            - containerPort: 3310
```

Comment:
Agreed - although for the recent scaling and probs configs most of that in-production testing was completed with manual production modifications.

For items that are truly environmentally modified, the value of the field can be set in the environment's values yaml - although I strongly urge against having port number change by environments.

As far as probe settings, same thing. The probe definition should be set in the parent helm chart - but if we find a need to tweak an individual value (memory, timeouts, etc) - we can set those in the values.yaml which would allow overriding in the environment values.yaml (see failureThreshold and other readiness probe settings in this PR)

Reply:
I like that approach, when/if the time comes, we can extract the values